src/methods/tx.rs

```diff
@@ -1,3 +1,29 @@
+//! This module allows you to query the status of a transaction signed by an account_id
+//!
+//! ## Example
+//!
+//! ```
+//! use near_jsonrpc_client::{methods, JsonRpcClient};
+//! use near_jsonrpc_primitives::types::transactions::TransactionInfo;
```

Comment:
`TransactionInfo` seems to be reexported from the `methods::tx` path. Let's self-contain the imports.